src/audit_simulation_sarah.py

```diff
@@ -0,0 +1,155 @@
+from src.multivariate_hypergeom_sample import multivhyper
+from src.brla_sarah import brla
+import numpy as np
+
+"""
+Class for simulating Bayesian RLA.
+Can be used on 2-candidate or multi-candidate elections with or without invalid votes.
+"""
+
+
+class auditsim:
+
+    def __init__(self, total_votes: int, vote_dist, audit_tier: int, invalid: bool = False):
+        """
+        Create audit simulation object.
+
+            @param total_votes: Total number of ballots cast in election.
```

Comment:
Must this be the sum of vote_dist? Should that be error-checked? Or calculated, and dropped as a parameter?